docs/swagger/spec.yml

```diff
@@ -2331,7 +2447,73 @@ definitions:
       maxModifiedDatetime:
         type: string
         format: datetime
+  UploadCreate:
+    type: object
+    properties:
+      uploadStatus:
+        type: string
+        description: Status of upload
+        enum:
+          - CREATED
+          - UPLOADING
+          - UPLOADED
+          - QUEUED
+          - PROCESSING
+          - COMPLETE
+      files:
+        type: array
+        items:
+          type: string
+      uploadType:
+        type: string
+        description: Source of files and uploads
+        enum:
+          - DROPBOX
+          - S3
+          - LOCAL
+      fileType:
+        type: string
+        description: type of data being uploaded, limited to two options right now
+        enum:
+          - GEOTIFF
+          - GEOTIFF_WITH_METADATA
+      datasource:
+        type: string
+        description: datasource of tiffs being uploaded
+        format: uuid
 
+  Upload:
+    allOf:
+    - $ref: '#/definitions/BaseModel'
+    - $ref: '#/definitions/UserTrackingMixin'
+    - $ref: '#/definitions/UploadCreate'
+    - type: object
+      properties:
```

Comment:
Huh, I never noticed that.

I think it should not be a child of `allOf`, and actually the `type: object` part seems unecessary too as that may be implied (the [uber example](https://github.com/OAI/OpenAPI-Specification/blob/master/examples/v2.0/yaml/uber.yaml#L196) doesn't use it and experimenting with ours shows it is unecessary.

What do you think we should do? I'll definitely move properties out from under `allOf` though -- that seems correct